packages/grid/_modules_/grid/models/api/gridApiRef.ts

```diff
@@ -1,7 +1,19 @@
 import * as React from 'react';
-import { GridApi } from './gridApi';
+import { GridApiCommon, GridApiCommunity, GridApiPro } from './gridApi';
 
 /**
- * The apiRef component prop type.
+ * The reference storing the api of the grid.
+ * TODO v6: Make the `GridApi` generic mandatory. This interface should only be used when accessing dynamically api methods or state based on the plan.
  */
-export type GridApiRef = React.MutableRefObject<GridApi>;
+export type GridApiRef<GridApi extends GridApiCommon = GridApiCommunity> =
+  React.MutableRefObject<GridApi>;
+
+/**
+ * The reference storing the api of the Community-plan grid.
+ */
+export type GridApiRefCommunity = React.MutableRefObject<GridApiCommunity>;
```

Comment:
This would be an equivalent, right?

```ts
export type GridApiRefCommunity = GridApiRef<GridApiCommunity>;
```

Is there any difference, or is it just matter of preference?

Reply:
When I did this line, `GridApiRef` did not have its generic yet
But you are right, it's better that way :+1: